openshift_images/using_images/using-images-overview.adoc

```diff
@@ -0,0 +1,31 @@
+[id="using-images-overview"]
+= Using images overview
+include::modules/common-attributes.adoc[]
+:context: using-images-overview
+toc::[]
+
+Use the following topics to discover the different Source-to-Image (S2I),
+database, and other container images that are available for {product-title}
+users.
+
+Red Hat’s official container images are provided in the Red Hat Registry at
+link:registry.redhat.io[registry.redhat.io]. {product-title}’s supported S2I, database,
+and Jenkins images are provided in the openshift4 repository in the Red Hat
+Registry. For example, `registry.redhat.io/openshift4/ose` for the
+OpenShift Application Platform image.
```

Comment:
In 4.x, ose image is from payload, just like jenkins image,  but other images are from  registry.redhat.io/openshift/
$ oc image info quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:6ccfbed9c79edafdbbee6bb9e6219576780fd1d6c28ba5bbc02c8b477aca8aa4 |grep name
             io.k8s.display-name=OpenShift Container Platform RHEL 7 Base
             name=openshift/ose-installer

Reply:
Should this be changed entirely to something similar to:

"Red Hat’s official container images are provided in the Red Hat Quay Registry at
link:quay.io[quay.io]."

Or

"Red Hat’s official container images are provided in the Red Hat Registry at
link:registry.redhat.io[registry.redhat.io]. {product-title}’s supported S2I, database,
and Jenkins images are provided in the openshift4 repository in the Red Hat Quay
Registry. For example, `quay.io/openshift4/ose` for the
OpenShift Application Platform image."

Reply:
Sorry, miss your info for a long time before, should be the first one. now we using installer image to install ose, and installer image from quay.io, which is integrated to payload.  here's example:
$ oc adm release info --pullspecs registry.svc.ci.openshift.org/ocp/release:4.1.0-rc.9 | grep -w "installer"
  installer                                     quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:6ccfbed9c79edafdbbee6bb9e6219576780fd1d6c28ba5bbc02c8b477aca8aa4
$ oc image info quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:6ccfbed9c79edafdbbee6bb9e6219576780fd1d6c28ba5bbc02c8b477aca8aa4 |grep name
             io.k8s.display-name=OpenShift Container Platform RHEL 7 Base
             name=openshift/ose-installer

Reply:
The second description is more better. But jenkins releted images in the payload are with hash imageid not name, like this  quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:6ccfbed9c79edafdbbee6bb9e6219576780fd1d6c28ba5bbc02c8b477aca8aa4 . quay.io/openshift4/ose  is invalid address, do not exist.